lib/SVF-FE/LLVMModule.cpp

```diff
@@ -145,10 +148,64 @@ void LLVMModuleSet::build()
             BasicBlock *exitBB =  const_cast<BasicBlock*>(LLVMUtil::getFunExitBB(func));
             svffun->setReachableBBs(reachableBBs);
             svffun->setExitBB(exitBB);
```

Comment:
make sure exitBB in SVFFun is a constant one.

Reply:
exitBB in SVFFun could not be a constant field since we cannot initiate it when building the SVFFun.